cmd/podman/shell_auto_completion.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var _bashCompletionCommand = &cobra.Command{
+	Use:   "bash-completion",
+	Short: "Generates shell script for bash shell",
+	Long: `To load completion run
```

Comment:
Can you add this as a hidden command.  We don't want to document this, and really do not intend users to use it.